.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!--
```

Comment:
I liked the Checklist idea and it also could be incorporated if all agree in the standard template and customized according to the project. However, it is missing questions/sections which are highly important and are found in the PR Template used by the other projects and in the standards. Following few argumentations. 

- The biggest part of the PR without this template has not the info WHAT, and WHY, and HOW and, and STEPS and just move forward because are reviewed for par team-mates
- If an issue be faced this information is helpful in order to investigate the changes performed in the PRs merged which could introduce the issue
- We need to keep in mind that our goal is to build a community and receive PRs from people which are not working with us and not attend our meetings and etc..
- The sections make clear what is expected
- The template makes clear that is expected direct and short answers and not that big descriptions found when the template is not applied which usually do not contain the required data.
- The PR owner will revalidate the PR when it will be filled which will be useful to find issues and/or changes that need be done before asking the review since the suggested questions will make the owner think more about it to clarify the achievement and steps to test it.
- The field to check that the PR was tested by the owner and reviewer push the people to do the tests and avoid we merge issues.